apps/voip/3cx/restapi/mode/system.pm

```diff
@@ -96,8 +96,8 @@ sub new {
     $self->{version} = '1.0';
     $options{options}->add_options(arguments => {
         "unknown-status:s"  => { name => 'unknown_status', default => '' },
-        "warning-status:s"  => { name => 'warning_status', default => '' },
-        "critical-status:s" => { name => 'critical_status', default => '%{health} =~ /false/' },
+        "warning-status:s"  => { name => 'warning_status', default => '%{display} =~ /HasUpdatesAvailable/ && %{health} =~ /true/' },
+        "critical-status:s" => { name => 'critical_status', default => '%{display} !~ /HasUpdatesAvailable/ && %{health} =~ /false/' },
```

Comment:
Not by default. BUt you can set the threshold in your command line.

Reply:
The previous `critical-status` would then trigger if no update is available, as `HasUpdatesAvailable` would then be false.

Would I then have to rename `HasUpdatesAvailable` to `SystemIsUptodate` ?
An available update would then trigger a critical alert.

Reply:
Yes that better. like that, we use the same for 'true' and 'false'.

Reply:
OK, let me improve this then :) :+1:

Reply:
Should be OK now Quentin :)